tests/phpunit.xml

```diff
@@ -1,13 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="../vendor/autoload.php">
-
-    <logging>
-        <log type="coverage-clover" target="../build/logs/clover.xml"/>
-    </logging>
-
-    <filter>
-        <whitelist>
-            <directory>../src/</directory>
-        </whitelist>
-    </filter>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="../vendor/autoload.php" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+  <coverage>
+    <include>
+      <directory>../src/</directory>
```

Comment:
Shouldnt this be normal `src/`?
No need for ../ etc

Reply:
I guess, it is relative path to the phpunit.xml file